src/utils/convertObjectToMap.js

```diff
@@ -0,0 +1,27 @@
+const convertObjectToMap = (gridFSObject) => {
+  if (gridFSObject && typeof gridFSObject === "object") {
+    if (Array.isArray(gridFSObject)) {
+      return gridFSObject.map(convertObjectToMap);
+    }
+    const convertedObject = {};
+
+    for (const key in gridFSObject) {
+      if (key === "pages" || key === "frames" || key === "nodes") {
+        const convertedMap = new Map();
+
+        for (const [pageId, pageNodes] of Object.entries(gridFSObject[key])) {
+          convertedMap.set(pageId, convertObjectToMap(pageNodes));
+          convertedMap[pageId] = convertObjectToMap(pageNodes);
+        }
+
+        convertedObject[key] = convertedMap;
+      } else {
+        convertedObject[key] = convertObjectToMap(gridFSObject[key]);
+      }
+    }
+    return convertedObject;
+  }
+  return gridFSObject;
+};
```

Comment:
- `P1`
`for...in`을 사용할때 `hasOwnProperty`를 같이 사용하면 좋을 것 같습니다!
추가적으로 컨벤션만 수정 부탁드립니다

```suggestion
const convertedObject = {};
const convertObjectToMap = (gridFSObject) => {
  if (gridFSObject && typeof gridFSObject === "object") {
    if (Array.isArray(gridFSObject)) {
      return gridFSObject.map(convertObjectToMap);
    }

    for (const key in gridFSObject) {
      if (key === "pages" || key === "frames" || key === "nodes") {
        const convertedMap = new Map();

        for (const [pageId, pageNodes] of Object.entries(gridFSObject[key])) {
          convertedMap.set(pageId, convertObjectToMap(pageNodes));
          convertedMap[pageId] = convertObjectToMap(pageNodes);
        }

        convertedObject[key] = convertedMap;
      } else {
        convertedObject[key] = convertObjectToMap(gridFSObject[key]);
      }
    }
    
    return convertedObject;
  }
  
  return gridFSObject;
};
```